api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -73,15 +60,11 @@ public class LeonardoApiClientImpl implements LeonardoApiClient {
 
   // Keep in sync with
   // https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala#L162
-  private static Set<LeonardoRuntimeStatus> STOPPABLE_RUNTIME_STATUSES =
-      ImmutableSet.of(
-          LeonardoRuntimeStatus.RUNNING,
-          LeonardoRuntimeStatus.STARTING,
-          LeonardoRuntimeStatus.UPDATING);
+  private static Set<ClusterStatus> STOPPABLE_RUNTIME_STATUSES =
```

Comment:
Do we still need to maintain this, or can we use the `runtimeModels` values directly now?